sorc/chgres_cube.fd/atm_input_data.F90

```diff
@@ -3018,6 +3018,14 @@ subroutine read_winds(u,v,localpet,octet_23,rlevs,lugb,pdt_num)
      print*, "- CALL CALCALPHA_ROTLATLON with center lat,lon = ",latin1,lov
      call calcalpha_rotlatlon(lat,lon,latin1,lov,alpha)
 
+   elseif (gfld%igdtnum == 1) then ! grid definition template number - non-E stagger rotated lat/lon grid
+
+     latin1 = real(float(gfld%igdtmpl(20))/1.0E6, kind=esmf_kind_r4)
```

Comment:
I would double check how `lov` is set. In template 32769 it is the longitude of the center of the grid. In template 1, it is the longitude of the south pole of projection. In `model_grid`, I had to add 90 degrees.

Reply:
Sorry, I meant `latin1`